src/api/app/models/workflow/step/branch_package_step.rb

```diff
@@ -44,7 +44,7 @@ def create_branched_package
     begin
       BranchPackage.new({ project: source_project_name, package: source_package_name,
                           target_project: target_project_name,
-                          target_package: target_package_name }).branch
+                          target_package: target_package_name(short_commit_sha: true) }).branch
```

Comment:
This is wrong. It will create the package with name  containing the short commit SHA. It's only the report which needs to change, so the ReportoScmJob as I pointed out in my previous comment.